community-membership.md

```diff
@@ -157,5 +158,69 @@ The following apply to the subproject(repos) for which one would be an owner:
+- Directly contributed to the subarea through implementation and / or review
+- Must have been active for 3 months or more for the given sub-area
+- Inactivity for more than 3 months leads to a removal vote by the repo maintainers and not an automatic removal
+
+
+### Acceptance
+
+  New Feature-Maintainers can be added to the project by a super-majority (two-thirds / 66.66%) vote. Only the maintainers of the repository in which the nomination is applied have a binding vote, while maintainers from other repositories are on an informed basis via a separate email thread.
+
+  A potential feature-maintainer may be nominated by an existing maintainer from the repository in which the nomination is applied to. A vote is conducted in private between the current maintainers over the course of a one week voting period. At the end of the week, votes are counted, and a pull request is made on the repo adding the new feature-maintainer to the feature-maintainers.md file.
+
+  Feature-maintainers MUST remain active. If they are unresponsive for >3 months, they will be automatically removed unless a super-majority of the other repository maintainers agrees to extend the period to be greater than 3 months.
+
+  A feature-maintainer may step down by submitting an issue stating their intent.
+
+### Example of Feature Maintainers in DAPR Runtime repo:
+Sub Area
+Person Name
+Workflow
+Chris Gillum
+Control Plane Service(mTLS)
+Josh V
+
+| **Subarea**   | **Name**                                  |
+| ---------- | ---------------------------------------------| 
+| Workflow    | Chris Gilum                                 | 
+| Control Plane Service(mTLS)| Josh V |
+
+### Responsibilities and privileges
+
+The following applies to the sub area(ex: building block API/ Control plane service/ Component provider) for which the feature-maintainer would be an owner:
+
+- Make and approve technical design decisions for the sub area
+- Set technical direction and priorities for the sub area.
+- Define milestones and releases working along with the maintainers
+- Decides on when PRs are merged to control the release scope
+- Mentor and guide approvers and contributors of the sub area
+- Escalate approver and maintainer workflow concerns. (For example responsiveness, availability, and general contributor community health) to the STC
+- Ensure continued health of sub area:
+- Adequate test coverage to confidently release
+- Tests are passing reliably (not flaky) and are fixed when they fail.
+- Work with other feature-maintainers & Core-maintainers to maintain the project's overall health and success holistically.
+- Membership tracked in feature-maintainer.md entry and scoped to a sub area.
+- MUST maintain components, review, and approve proposals for enhancing areas falling under the user sub area.
+- Actively participate in issue triages and PR reviews
+- Set milestone priorities for the sub areas or delegate the responsibility to repo maintainers.
```

Comment:
Please, remove the delegate part. Maintainers will set priorities for the project overall and it might or might not include a specific area.

Reply:
> Please, remove the delegate part. Maintainers will set priorities for the project overall and it might or might not include a specific area.

Agreed, changed it as to set the priorities working with Repo maintainers